ambry-commons/src/main/java/com/github/ambry/clustermap/HelixVcrUtil.java

```diff
@@ -43,10 +41,34 @@
  */
 public class HelixVcrUtil {
   private static final Logger logger = LoggerFactory.getLogger(HelixVcrUtil.class);
-  public static final List<String> ignoreResourceKeyWords = Arrays.asList("aggregation", "trigger", "stats");
   static final String SEPARATOR = "/";
   private static final ZNRecordSerializer ZN_RECORD_SERIALIZER = new ZNRecordSerializer();
   private static final String DELAYED_REBALANCER_CLASS_NAME = "DelayedAutoRebalancer";
+  private static final Set<String> smallIntegers = new HashSet<>();
+
+  static {
+    for (int i = 0; i < 128; i++) {
+      smallIntegers.add(String.valueOf(i));
+    }
+  }
+
+  /**
+   * Return {@code true} is the given resource name is a resource that contains partition information.
+   * @param resourceName The resource name.
+   * @return {@code true} when the given resource contains partition inforrmation
+   */
+  public static boolean isPartitionResourceName(String resourceName) {
+    // all the paritions are placed under resource named as integer values.
```

Comment:
nit: paritions -> partitions

The purpose of smallIntegers is to create a fast path to avoid needing to parse the string as an int in most cases?

Reply:
yes, I only check one cluster and it has 117 resources, so I set the value to be 128, but I checked the other two clusters, they are both more than 128 and less than 512, I will set the maximum size to 512, this would cover all of them.